docs/modules/globalfee.md

```diff
@@ -1,26 +1,28 @@
 # Gaia Fees and Fees Checks
 
 ## Fee Parameters
-The CosmosHub allows managing fees using 3 parameters:
+The CosmosHub allows managing fees requirement using 4 parameters. At the network level, there are three parameters from globalfee modules (`MinimumGasPricesParam`, `BypassMinFeeMsgTypes`, and `MaxTotalBypassMinFeeMsgGasUsage`) that can be set by gov proposal. Additionally, there's a fourth parameter that enables individual nodes to impose supplementary fee amount requirement.
 
-1. setting global fees (`MinimumGasPricesParam`)
-Global fees are defined at the network level by setting `MinimumGasPricesParam`, via [Gov Proposals](https://hub.cosmos.network/main/governance/proposals/)
-
-2. `minimum-gas-prices`
-This parameter is part of the node configuration, it can be set in the `config/app.toml` configuration file.
-
-3. `bypass-min-fee-msg-types`
-This parameter is part of the node configuration, it can be set in the `config/app.toml` configuration file.
-This represents a list of message types that will be excluded from paying any fees for inclusion in a block.
-
-Both global fees (`MinimumGasPricesParam`) and `minimum-gas-prices` represent a list of coins, each denoted by an amount and domination as defined by [sdk.DecCoins](https://github.com/cosmos/cosmos-sdk/blob/82ce891aa67f635f3b324b7a52386d5405c5abd0/types/dec_coin.go#L158) 
+1. Global fees (`MinimumGasPricesParam`).\
+Global fee `MinimumGasPricesParam` is established at the network level through global fee params setting via Governance Proposals, and it sets the fee requirements that the entire network must adhere to.
+2. `minimum-gas-prices` in `app.toml`\
+   By adjusting the minimum-gas-prices parameter in app.toml, nodes can enforce a fee that is higher than the globally defined MinimumGasPricesParam. However, it's important to note that this configuration solely determines whether transactions are eligible to enter this specific node's mempool.
 
+3. `bypass-min-fee-msg-types` and `max_total_bypass_min_fee_msg_gas_usage`.\
+ These two parameters are also part of the global fee params from gaiad v11.0.0, and can be changed through Gov Proposals. `bypass-min-fee-msg-types` represents a list of message types that will be excluded from paying any fees for inclusion in a block, `max_total_bypass_min_fee_msg_gas_usage` is the limit of `bypass-min-fee-msg-types` gas usage.
 
 ## Concepts
 
 ## Global Fees
 
-Global fees consist of a list of `sdk.DecCoins` e.g., `[1uatom, 2stake]`. 
+Global Fees module has three params that can be set by gov proposal `param-change`: 
+- `MinimumGasPricesParam`
+- `BypassMinFeeMsgTypes` 
+- `MaxTotalBypassMinFeeMsgGasUsage`
+
+### Global Fee Params: `MinimumGasPricesParam`
+
+Global fees consist of a list of [`sdk.DecCoins`](https://github.com/cosmos/cosmos-sdk/blob/82ce891aa67f635f3b324b7a52386d5405c5abd0/types/dec_coin.go#L158).
```

Comment:
**Network level,** global fees consist of a list of [`sdk.DecCoins`](https://github.com/cosmos/cosmos-sdk/blob/82ce891aa67f635f3b324b7a52386d5405c5abd0/types/dec_coin.go#L158).